testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -60,6 +60,10 @@ class DockerComposeNamespace(DockerComposeBaseNamespace):
         COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.yml",
         COMPOSE_FILES_PATH + "/storage-proxy/docker-compose.storage-proxy.testing.yml",
     ]
+    LEGACY_V3_CLIENT_FILES = [
+        COMPOSE_FILES_PATH
+        + "/extra/integration-testing/test-compat/docker-compose.compat-mender-3.6.yml",
```

Comment:
These are the files we have for the compatibility test. Actually the file was missing after Mender 3.6 release and I issued an independent PR [here](https://github.com/mendersoftware/integration/pull/2506) (and brought into our feature branch [here](https://github.com/mendersoftware/integration/pull/2515)). I am just reusing the file instead of creating a new one specifically for this test.